### DISCUSSION TITLE
Error when following the tutorial in Gromacs, example 1

### DISCUSSION THREAD

**Discussion by xih72** [SIRAHFF/documentation#76] [Q&A] [answered]
I was following the tutorial of SIRAH in Gromacs: 
https://sirahff.github.io/documentation/Tutorials%20gromacs.html
The 1st example, 1. DNA molecule in explicit solvent
1.4. Run the simulation
Equilibration: 
gmx grompp -f ../sirah.ff/tutorial/1/GPU/eq_CGDNA.mdp -p ../topol.top -po eq.mdp -n ../dna_cg_ion.ndx -c dna_cg_em.gro -o dna_cg_eq.tpr
Here I got the error message: 
"""Fatal error:
Cannot find position restraint file restraint.gro (option -r).
From GROMACS-2018, you need to specify the position restraint coordinate files
explicitly to avoid mistakes, although you can still use the same file as you
specify for the -c option."""

This restraint.gro file was not mentioned before this step. What's going on?

**Comment by luciannahss** [comment, marked as answer]
The tutorial used a old version of GROMACS. If you use GROMACS 2018 or even newer version, "-r" flag should be typed explicitly in the "gmx grompp" command. 
Thus, you just need to repeat the same .gro file as the -c flag, as the error message explains. 
Something like:
gmx grompp -f ../sirah.ff/tutorial/1/GPU/eq_CGDNA.mdp -p ../topol.top -po eq.mdp -n ../dna_cg_ion.ndx -c dna_cg_em.gro -r dna_cg_em.gro -o dna_cg_eq.tpr

Further details of how to use 'gmx grompp' in GROMACS 2018 including the '-c' and '-r' flags can be found here:
http://manual.gromacs.org/documentation/2018/onlinehelp/gmx-grompp.html

Let us know if it works.

**Comment by xih72** [comment]
Thanks! Since it's 2024 now, I doubt any user would use a version of Gromacs older than the 2018 version. it's better to modify the tutorial. You can close this ticket after doing so.
